src/components/about/sections/PersonDisplay.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Col } from "react-bootstrap";
+import "../index.css";
+// i18n translations might still be loaded by the http backend
+// use react's Suspense
+export default function PersonDisplay(props) {
+  const { t } = useTranslation();
+  let photo;
+  // manual picture setting
+  switch (props.name) {
+    case "nasser":
+      photo = require("././images/staff0.png");
+      break;
+    case "jawdat":
+      photo = require("././images/staff1.png");
+      break;
+    case "turgut":
+      photo = require("././images/staff2.png");
```

Comment:
it seems to me that if you're going to do a switch-case here you might as well move this into peopleOrdering, in which case i would call it `peopleSpec` (spec is slightly different from data in implication)